src/orders/components/OrderTransaction/components/TransactionEvents/TransactionEvents.tsx

```diff
@@ -31,93 +26,57 @@ const useStyles = makeStyles(
         },
       },
     },
-    hover: {
-      backgroundColor: theme.palette.saleor.active[5],
-    },
-    colSmall: {
-      [theme.breakpoints.down("md")]: {
-        // Take as little space as possible on mobile
-        width: "1%",
-        whiteSpace: "nowrap",
-      },
-    },
-    colStatus: {
-      [theme.breakpoints.up("md")]: {
-        // Max text with "Success"
-        width: "126px",
-      },
-    },
-    colPspReference: {
-      [theme.breakpoints.up("md")]: {
-        width: "250px",
-      },
-    },
-    colLast: {
-      // Align with card
-      [theme.breakpoints.up("md")]: {
-        "&&&": {
-          paddingRight: "32px",
-          width: "35%",
-          textAlign: "right",
-        },
-      },
-      [theme.breakpoints.down("md")]: {
-        whiteSpace: "nowrap",
-      },
+    noEvent: {
+      color: theme.palette.saleor.main[2],
     },
   }),
   {
     name: "OrderTransactionEvents",
   },
 );
 
+const isFakeEventsList = (
+  events: TransactionEventFragment[] | TransactionFakeEvent[],
+): events is TransactionFakeEvent[] =>
+  events[0]?.__typename === "TransactionFakeEvent";
```

Comment:
question: shouldn't you check if you have at least one element in array ?

Reply:
nope, this is enough - if the array is empty `events[0]` returns undefined and `undefined?.__typename` is also undefined and this definitely doesn't equal `TransactionFakeEvent` string ;)

Reply:
This is used to check the evetns array + as a custom type guard, if the array is empty this doesn't change anything because in such a case we won't render `EventItem`